rxdrive/src/main/java/com/francescocervone/rxdrive/RxDrive.java

```diff
@@ -462,6 +462,37 @@ public DriveFolder getAppFolder() {
             }
         }).subscribeOn(Schedulers.io());
     }
+    
+    /**
+     * Updates a file on Drive
+     *
+     * @param driveFile drive file
+     * @param content the content to write
+     * @return an Observable with the DriveId
+     */
+    public Observable<DriveFile> updateFile(final DriveFile driveFile, final ByteArrayInputStream content) {
+        return Observable.defer(new Func0<Observable<DriveFile>>() {
+            @Override
+            public Observable<DriveFile> call() {
+                DriveApi.DriveContentsResult driveContentsResult = driveFile
+                        .open(getGoogleApiClient(), DriveFile.MODE_WRITE_ONLY, null)
```

Comment:
`getGoogleApiClient()` method does not exist. You should use `mClient` field variable